### ISSUE TITLE
Need to install pexpect for otl866

### ISSUE THREAD

**Issue by RobertBaruch** [ProgHQ/open-tl866#66]

**Comment by JohnDMcMaster** [comment]
Can you clarify the issue? You mean this is a hard requirement to fulfill on Windows?

**Comment by RobertBaruch** [comment]
No, I mean pexpect as a prerequisite it isn't in the documentation and wasn't installed by setup.py, and users would just get an error if they run otl866. Or is this something that setup.py can do?
